app/root/root.css

```diff
@@ -1066,6 +1060,7 @@ code .comment {
   background-color: transparent;
   cursor: pointer;
   padding: 12px;
+  margin: -12px 0;
```

Comment:
nit: is there a way to do this without using negative margins?

Reply:
Its margin to offset the padding, while keeping the click targets large. We can reduce click target size - but I'd rather keep them big for the occasional mobile user.